docs/modules/getting_started/partials/proc_connecting-to-host.adoc

```diff
@@ -0,0 +1,24 @@
+= Access services from host to {prod}
+
+[NOTE]
+====
+**Perform this procedure when using user mode network which is default for Mac/Windows.**
+====
+
+If you have a service which is running on the host and want to consume it with {prod}
+
+. Ensure that you have enabled the proper config value
++
+----
+$ crc config set network-mode user
+$ crc config set host-network-access true
```

Comment:
maybe explain the default mode or default config or how to check / verify it'll work